src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Object getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        ball.setColor(randomColor);
+        ball.setNumber(random.nextInt(100));
+        return ball;
+    }
+}

```

Comment:
The method `getRandomBall` should not return an Object type. It should return a `Ball` type since that is what is being created and returned. Using a specific type instead of a generic `Object` provides type safety and reduces the need for type casting by the caller.